src/main/java/com/amazon/opendistroforelasticsearch/sql/esdomain/LocalClusterState.java

```diff
@@ -35,11 +35,7 @@
 import org.json.JSONObject;
 
 import java.io.IOException;
-import java.util.Arrays;
-import java.util.Collection;
-import java.util.List;
-import java.util.Map;
-import java.util.Objects;
+import java.util.*;
```

Comment:
Please do not use star imports, it makes it more difficult to quickly identify where each of the used types is coming from. You can update IntelliJ's setting to not auto-convert to star.

Reply:
will fix this and the similar issue in other files. Thanks.